### PR TITLE
[Round 4 Method Reference Static]Lps 182411 1

### PR DESCRIPTION
**Pull request by slnn**
https://issues.liferay.com/browse/LPS-182411

### CODE REVIEW

File: modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/test/java/com/liferay/portal/search/tuning/rankings/web/internal/util/RankingResultUtilTest.java [hunk 1]
```diff
@@ -51,15 +60,55 @@ public class RankingResultUtilTest extends BaseRankingsWebTestCase {
...
+	public static void tearDownClass() {
+		_frameworkUtilMockedStatic.close();
+	}
+
 	@Before
 	public void setUp() throws Exception {
+		RankingResultUtil rankingResultUtil = new RankingResultUtil();
+
+		Snapshot<DocumentBuilderFactory> documentBuilderFactorySnapshot =
+			Mockito.mock(Snapshot.class);
```
Comment (dantewang) [inline modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/test/java/com/liferay/portal/search/tuning/rankings/web/internal/util/RankingResultUtilTest.java:84]: @slnn Hmmm maybe I missed this thing in your previous change. Is it possible to NOT mock `Snapshot`? You can register the mocked "_documentBuilderFactory" or "_searchResultInterpreterProvider" using the bundle context, and expect that the snapshot can properly return them.

**Review by dantewang — CHANGES_REQUESTED** [review]

**Comment by dantewang** [comment]
@slnn Please update the test and resend. Could you please also review your other merged changes to and send a follow-up PR to update the tests you previously modified? Thanks.

**Comment by dantewang** [comment]
ci:stop

**Comment by slnn** [comment]
Resend in https://github.com/liferay-core-infra/liferay-portal/pull/4383

**Pull request closed without merging**